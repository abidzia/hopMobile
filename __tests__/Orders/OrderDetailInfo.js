import "react-native";
import React from "react";

// Note: test renderer must be required after react-native.
import renderer from "react-test-renderer";

import OrderDetailInfo from "../../src/components/Orders/OrderDetailInfo";

it("renders correctly", () => {
  order = {
    order: "123456",
    order_type: "Delivery",
    delivery_address: "Address",
    user_apartment: "user_apartment",
    user_cross_street: "user_cross_street",
    user_phone: "123456789",
    time: "12:00pm",
    confirmation_code: "123456",
    instructions: "instructions",
    customer_name: "customer_name"
  };
  const tree = renderer.create(<OrderDetailInfo order={order} />);
  expect(tree).toMatchSnapshot();
});
