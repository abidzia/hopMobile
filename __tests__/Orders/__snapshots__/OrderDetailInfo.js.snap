// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "padding": 20,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "black",
            "fontSize": 32,
          }
        }
      >
        #
        123456
         
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "grey",
            "fontSize": 32,
          }
        }
      >
        Delivery
      </Text>
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "grey",
          "fontSize": 22,
        }
      }
    >
      Received 
      12:00pm
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "grey",
          "fontSize": 22,
          "paddingTop": 20,
        }
      }
    >
      Confirmation Code
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "black",
          "fontSize": 30,
          "fontWeight": "100",
        }
      }
    >
      123456
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "black",
          "fontSize": 20,
          "paddingTop": 20,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "fontWeight": "bold",
            },
          ]
        }
      >
        Special Instructions: 
      </Text>
      instructions
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "grey",
          "fontSize": 20,
          "fontWeight": "100",
        }
      }
    />
  </View>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "black",
        "flex": 1,
        "fontSize": 22,
        "paddingTop": 10,
        "textAlign": "right",
      }
    }
  >
    customer_name
    (123) 456 789
    Address
    user_apartment
    user_cross_street
  </Text>
</View>
`;
